OpenDreamRuntime/Procs/Native/DreamProcNative.cs

```diff
@@ -8,6 +11,50 @@ static class DreamProcNative {
             DreamProcNativeRoot.DreamManager = IoCManager.Resolve<IDreamManager>();
 
             DreamObjectDefinition root = objectTree.GetObjectDefinition(DreamPath.Root);
+
+            Assembly asm = Assembly.GetExecutingAssembly();
+            
+            var q = from t in Assembly.GetExecutingAssembly().GetTypes()
+                    where t.Name == "DreamProcNativeRoot" && t.IsClass && t.Namespace == "OpenDreamRuntime.Procs.Native"
+                    select t;
+
+            foreach (var t in q)
```

Comment:
There's a more direct way to get the type of a class, `typeof(DreamProcNativeRoot)`